vm/chef/cookbooks/espocrm/files/espocrm-utils

```diff
@@ -81,19 +81,20 @@ function change_php_config() {
 #######################################
 function install_app() {
   local -r curl="curl --request POST -c /tmp/cookie -b /tmp/cookie"
+  local -r URL="http://localhost/install/"
   echo "$URL"
```

Comment:
You can remove this echo, not that useful.

Reply:
Done